<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wnd_main</class>
 <widget class="QMainWindow" name="wnd_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curso Python</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_ci">
       <property name="text">
        <string>Cédula</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_ci"/>
     </item>
     <item>
      <widget class="QLabel" name="lbl_name">
       <property name="text">
        <string>Nombres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_name"/>
     </item>
     <item>
      <widget class="QLabel" name="lbl_gender">
       <property name="text">
        <string>Género</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_gender">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>481</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_search">
       <property name="text">
        <string>Cédula</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_search"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
